ui/imports/shared/panels/ProfileBioSocialsPanel.qml

```diff
@@ -84,13 +85,27 @@ Control {
     contentItem: ColumnLayout {
         spacing: 20
 
-        StatusBaseText {
+        StatusScrollView {
+            id: scrollView
+            visible: root.bio !== ""
```

Comment:
small suggestion: `visible: root.bio` is valid condition for this :)

Reply:
fixed